src/deluge/modulation/arpeggiator.cpp

```diff
@@ -138,13 +133,13 @@ void ArpeggiatorForDrum::noteOff(ArpeggiatorSettings* settings, ArpReturnInstruc
 	}
 
 	arpNote.velocity = 0; // Means note is off
+	reset();
```

Comment:
What's the purpose of this reset?

Reply:
Oh i see what you mean, that function does nothing here. Removed.